packages/delegation-deployments/script/validate-contract-deployments.ts

```diff
@@ -204,17 +231,22 @@ const allChainsDone = chainIds.map(async (chainIdAsString) => {
         console.error(
           `${chain.name}: ${contractName} is not deployed at ${contractAddress}`,
         );
-        hasFailed = true;
+        hasThisChainFailed = true;
       }
     } catch (error) {
       console.error(`RPC Request failed for ${chain.name}: ${contractName}`);
```

Comment:
The catch block logs a generic message without including the actual error details, which hides the root cause of failures such as network timeouts, rate limiting, or authentication errors.

Reply:
Yeah, I agree that's annoying, but it's generally the same thing: rate limiting, and the fix is normally the same: try again 🤣.

I didn't want to pollute the output with error details, because it distracts from what we're really wanting: is the contract deployed on all chains.